deepl-connector-test/src_test/com/axonivy/connector/deepl/test/DeepLTest.java

```diff
@@ -70,5 +73,37 @@ public void document(BpmClient bpmClient) throws IOException{
     assertThat(translated).isNotNull();
     assertThat(translated.getJavaFile()).exists();
   }
+  
+  @Test
+  public void translateAdvanced(BpmClient bpmClient){
+    ExecutionResult result = bpmClient.start()
+      .subProcess(Start.TRANSLATE_ADVANCED)
+      .withParam("text", "Hello World")
+      .withParam("targetLanguage", getDefaultOptions())
+      .execute();
+    deepl.translate.Data data = result.data().last();
+    assertThat(data.getOutput()).isEqualTo("Hallo Welt");
```

Comment:
can you incorporate the 'html' tag handling into this test? 
So that we easily see the benefits from the advanced options?